src/dlsproto/client/README.rst

```diff
@@ -58,3 +58,12 @@ that there are two ways to assign some requests:
    blocking manner -- the current task will be suspended until the assigned
    request is finished.
 
+Controlling Requests
```

Comment:
I'm not sure this is generic enough to note here. Doesn't it only apply to GetRange?

Reply:
I've just added it here as it is a client's feature IMHO, so it's useful to be in the high-level documentation.

Reply:
It's kind of half and half. The basic idea of controlling a request is generic (and is mentioned in the swarm README: https://github.com/sociomantic-tsunami/swarm/blob/v5.x.x/src/swarm/README_client_neo.rst#request-controllers). The thing of the timeout is totally specific to GetRange. It ought to be mentioned at https://github.com/nemanja-boric-sociomantic/dlsproto-1/blob/1893a28a3677139eb7b3831451f58c1afdaa478d/src/dlsproto/client/request/GetRange.d#L3. Ah I see you don't have request documentation there, though. (dmqproto and dhtproto do.) Maybe something for a follow-up PR.

Reply:
Ah, right. Ok, I'll extend the docs in the follow-up PR.